src/components/FlightDetails/FlightItineraryInfo.tsx

```diff
@@ -0,0 +1,101 @@
+import React, {useEffect, useState} from "react";
+import {FlightResultsProps} from "../../pages/HomePage/interfaces";
+import {Box, Grid, LinearProgress, Typography} from "@mui/material";
+import {airlineFailoverLogo} from "../FlightCard/FlightInfoComponent";
+import flightCardStyles from "../FlightCard/FlightCard.module.css";
+import {convertDate, timeTravelDiff} from "../../utils/utils";
+import moment from "moment";
+import {getAirlineByCodes, IAirlineInfo} from "../../services/FlightDetails.service";
+
+type FlightItineraryInfoProps = {
+    flightOffer: FlightResultsProps,
+};
+
+const FlightItineraryInfo: React.FC<FlightItineraryInfoProps> = ({flightOffer}: FlightItineraryInfoProps) => {
+    const [airlines, setAirlines] = useState<Map<string, string> | null>(null);
+    const {passengers} = flightOffer;
```

Comment:
How are we getting this data?
I ran your branch locally and this field is empty. I tried running the latest version of the backend [dev](https://github.com/skywalkersdxc/skywalkersdxc-back/tree/dev) and also this branch: [FS-70](https://github.com/skywalkersdxc/skywalkersdxc-back/tree/FS-70-AirlineInfoEndpoints). But none of them returned the **passengers** field.

Reply:
Yeah, the endpoint doesn't return the passengers, but because of the required design I'm adding the field from the value we get in the passengers selection element over the home page.

Reply:
Can you verify if now you can see the passengers info?